app/controllers/sessions_controller.rb

```diff
@@ -4,16 +4,23 @@ def new; end
   def create
     user = User.find_by email: params[:session][:email].downcase
     if user&.authenticate(params[:session][:password])
-      log_in user
-      redirect_to user
+      create_controller user
     else
       flash.now[:danger] = t "controllers.sessions.invalid_infor"
       render :new
     end
   end
 
   def destroy
-    log_out
+    log_out if logged_in?
     redirect_to root_url
   end
+
+  private
+
+  def create_controller user
```

Comment:
```suggestion
  def login_user user
```
e chú ý đặt tên function, biến có nghĩa nhé

Reply:
À chết em lại quên huhu sorry chị